src/game/common/globaldata.h

```diff
@@ -63,6 +63,8 @@ class GlobalData : public SubsystemInterface
 
     bool Set_Time_Of_Day(TimeOfDayType time);
 
+    Utf8String Get_Path_User_Data() const { return m_userDataDirectory.Str(); }
```

Comment:
This seems wrong, it shouldn't be calling .Str() if its returning Utf8String.
Fix this and it should be good.

Reply:
Fixed.